luminoso_api/client.py

```diff
@@ -125,7 +125,7 @@ def _get_token_auth(username, password, token, token_file, root_url):
             if username is None:
                 username = os.environ['USER']
             if password is None:
-                password = getpass('Password for %s: ' % username)
+                password = getpass(('Password for %s: ' % username).encoding('utf8'))
```

Comment:
Yea sorry, "encoding" must have been a muscle memory typo. 

I tested "encode" locally, and it worked it both 2.7 & 3.5